mw-webapp/src/logic/waysTable/columns.tsx

```diff
@@ -21,21 +21,20 @@ export const OWNER_EMAIL = "Owner's email";
  */
 export const columns = [
   columnHelper.accessor<"uuid", string>("uuid", {
-    // Maybe we should add way's name? Need to think)
     header: "Way's uuid",
 
     /**
      * Cell with owner's name
      */
     cell: ({row}) => renderCellValue(row.original.uuid),
   }),
```

Comment:
Maybe we no need render uuid of way in the table

Reply:
I think Victor said that it might be there

Reply:
If you mean this video QA session https://www.youtube.com/watch?v=fLDg2SLUcWg

Then Viktor speak about it on 2.26-2.35 .

Reply:
If you think then we will need ID then you can don't delete, but then add comment for delete it later

Reply:
Thanks, deleted